src/components/QuantityContainer/QuantityContainer.js

```diff
@@ -0,0 +1,49 @@
+import { Button, makeStyles } from '@material-ui/core';
+import React, { useState } from 'react';
+
+const useStyles = makeStyles(() => ({
+  root: {
+    display: 'flex',
+    width: '150px',
+    justifyContent: 'space-evenly',
+    alignItems: 'center',
+    marginLeft: '5px',
+  },
+  actionBtn: {
+    borderRadius: '50%',
+    width: '40px',
+    height: '40px',
+    minWidth: 0,
+    border: '1px solid #0000001f',
+    margin: '0 5px',
+  },
+}));
+
+const QuantityContainer = () => {
+  const classes = useStyles();
+  const [quantity, setQuantity] = useState(1);
+
+  return (
+    <div className={classes.root}>
+      <Button
+        variant="contained"
+        color="primary"
+        className={classes.actionBtn}
+        onClick={() => setQuantity(quantity + 1)}
+      >
+        +
+      </Button>
+      <span className={classes.quantity}>{quantity}</span>
+      <Button
+        variant="contained"
+        color="primary"
+        className={classes.actionBtn}
+        onClick={() => quantity > 1 && setQuantity(quantity - 1)}
+      >
+        -
+      </Button>
```

Comment:
is it really necessary? what should be the reason for not using this?

Reply:
You wouldn't have to write custom CSS to make it round etc. That component is made for this kind of purpose in mind. Check the demo of it on their page.

Reply:
I don't think that's a strong point there to remove it, also if I use that, it would require adding custom Icons as well. I think we should go with this as rounding a button is not so much of a work.

Reply:
> it would require adding custom Icons

Plus and Minus icons are available on Material icons 🤔

Reply:
What's heavy? plus and minus text or the icons?

Reply:
Let's go with text then.

Reply:
Did you mean heavy in terms of size?

It makes no sense to use what it's not meant for.
We literally have Icon buttons for this purpose.